eva/executor/fuzzy_join_executor.py

```diff
@@ -0,0 +1,51 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Generator, Iterator
+
+from eva.executor.abstract_executor import AbstractExecutor
+from eva.executor.executor_utils import ExecutorError, apply_predicate
+from eva.models.storage.batch import Batch
+from eva.plan_nodes.fuzzy_join_plan import FuzzyNestedJoinPlan
+from eva.utils.logging_manager import logger
+
+
+class FuzzyJoinExecutor(AbstractExecutor):
```

Comment:
Maybe rename it to `NestedLoopJoin`

Reply:
Could have a base `NestedLoopJoin` class, and FuzzyJoin could be a special type of it